tools/scripts/dylan-lldb

```diff
@@ -1,35 +1,53 @@
 #! /bin/sh
 
+# Dylan LLDB helper script
+# This starts LLDB with some useful extensions preloaded
+# Usage: dylan-lldb [args...]
+# where [args...] are any LLDB arguments or options.
+
+# Relevant environment variables:
+# OPEN_DYLAN_LLDB - overrides the location of the lldb executable 
+#  Default is `lldb` on the path.
+# OPEN_DYLAN_LLDB_HELPER - overrides the location of the extension packages
+#  Default is ./share/opendylan/lldb/dylan relative to the OD installation firectory. For development use mainly
+# PYTHONPATH - this is used by LLDB itself to find the Python support packages
+#  On some systems it needs to be set manually, for example lldb-9 on Debian has PYTHONPATH=/usr/lib/llvm-9/lib/python3.7/site-packages
+#  Default is whatever `lldb -P` prints
+
```

Comment:
Great comments. It looks like `dylan-lldb` isn't documented in https://opendylan.org/documentation/getting-started-cli/debugging-with-gdb-lldb.html if you feel like remedying that. :)

Reply:
OK, done, see commits below!